hera_mc/cm_handling.py

```diff
@@ -39,6 +39,14 @@ class Handling:
 
     def __init__(self, args):
         self.args = args
+    #------START-TO-DO------#
+        #-#db = mc.connect_to_mc_db(self.args)
+        #-#self.session = db.sessionmaker()
+        #-#self.station_types = None
+
+    #-#def close(self):
+        #-#self.session.close()
+    #------END-TO-DO------#
```

Comment:
As I mentioned in my comment earlier, it would be good to be more selective when committing so as not to add these kinds of changes to `master`. But to get this turned around quickly, I am OK with this being here this time.